api/cluster.go

```diff
@@ -339,15 +340,17 @@ func (s *Server) indexList(ctx *middleware.Context, req models.IndexList) {
 }
 
 func (s *Server) getData(ctx *middleware.Context, request models.GetData) {
-	series, err := s.getTargetsLocal(ctx.Req.Context(), request.Requests)
+	var ss models.StorageStats
+	series, err := s.getTargetsLocal(ctx.Req.Context(), &ss, request.Requests)
 	if err != nil {
 		// the only errors returned are from us catching panics, so we should treat them
 		// all as internalServerErrors
 		log.Errorf("HTTP getData() %s", err.Error())
 		response.Write(ctx, response.WrapError(err))
 		return
 	}
-	response.Write(ctx, response.NewMsgp(200, &models.GetDataResp{Series: series}))
+	ss.Trace(opentracing.SpanFromContext(ctx.Req.Context()))
```

Comment:
I dont think this is needed here.  `ss.Trace()` is already called at the end of `s.getTargetsLocal()`

Reply:
makes sense. we only need it in executePlan (for individual nodes, both local and cluster peers), and executePlan (for aggregated-across-peers per-response stats)